adoc/Release-Notes.adoc

```diff
@@ -107,7 +107,20 @@ endif::[]
 [id='sec.1_5_1.issue']
 ==== Known Issues
 
-* On occasion, the `uaa` pod may fail to start due to database migration failures. In these situations, manual intervention is required to track the last completed transaction in the `uaadb` database, update the `schema_version` table with the record of the last completed transaction, then restart the migration. Please contact support for further instructions.
+[IMPORTANT] 
+====
+* For upgrades from {product} 1.5 to 1.5.1, if the `mysql` roles of `uaa` and `scf` are in high availability mode, it is recommended to first scale them to single availability. Performing an upgrade with these roles in HA mode runs the risk of encountering potential database migration failures, which will lead to pods not starting. This does not impact usage when connected to external databases.
```

Comment:
For clusters in a high availability (HA) configuration, the `mysql` roles of `uaa` and `scf` should be scaled down to single instances before upgrading. Upgrading with multiple instances of `mysql` running may cause database migration failures, which will lead to pods not starting after upgrade. Clusters using external databases (via `DB_EXTERNAL_*` settings) are unaffected.

Reply:
Changed, thanks!